src/core/services/TransactionService.ts

```diff
@@ -101,27 +101,36 @@ export class TransactionServiceImpl implements TransactionService {
   public handleTransaction = async ({
     network,
     tx,
+    useExternalService = false,
     renderNotification = true,
   }: HandleTransactionProps): Promise<TransactionReceipt> => {
     const { NETWORK_SETTINGS } = getConfig();
     const currentNetworkSettings = NETWORK_SETTINGS[network];
+    const useNotifyService = useExternalService && currentNetworkSettings.notifyEnabled;
+
     let updateNotification: UpdateNotification | undefined;
     let dismissNotification: () => void = () => undefined;
-    try {
-      if (renderNotification) {
-        if (currentNetworkSettings.notifyEnabled) {
-          notify.hash(tx.hash);
-        } else {
-          const { update, dismiss } = notify.notification({
-            eventCode: 'txSentCustom',
-            type: 'pending',
-            message: 'Your transaction has been sent to the network',
-          });
-          updateNotification = update;
-          dismissNotification = dismiss;
-        }
+    let notifyServiceFailed = false;
```

Comment:
Same here, this will be `undefined` and thus falsy
```suggestion
    let notifyServiceFailed;
```

Reply:
much nit?

changed here https://github.com/yearn/yearn-finance-v3/commit/cc4c6b9e0a0ee50a21a9454875983308ccc16afc XD